lib/scala/project-manager/src/main/scala/org/enso/projectmanager/service/ProjectService.scala

```diff
@@ -278,10 +278,12 @@ class ProjectService[
       openTime <- clock.nowInUtc()
       updated = project.copy(lastOpened = Some(openTime))
       _ <- repo.update(updated).mapError(toServiceFailure)
+      projectWithDefaultEdition =
+        project.copy(edition = Some(DefaultEdition.getDefaultEdition))
       sockets <- startServer(
         progressTracker,
         clientId,
-        updated,
+        projectWithDefaultEdition,
```

Comment:
So that's the gist of it, right? Ignore whatever edition is there and use the default one?

```
projectWithDefaultEdition =
        project.copy
```
shouldn't this be `updated.copy`? Not that it matters probably because the project has already been saved in the index.